### PR TITLE
chore: cleanup Notebooks empty state

### LINKED ISSUE #2521: Notebooks: Update the Notebooks list page empty state
We would like to update the Notebooks list empty state now that templates has been added. 

Design:

CSS to turn the existing image to grayscale: https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/grayscale()

Design: @juliajames12

### PR DESCRIPTION
**Pull request by joelfried**
remove create button from notebooks empty state, change image to grayscale, shrink component size to reduce unnecessary scrolling

Closes #2521

### CODE REVIEW

File: src/flows/components/FlowsIndexEmpty.tsx [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
 import 'src/flows/components/EmptyPipeList.scss'
 
 const FlowsIndexEmpty = () => {
   return (
-    <EmptyState size={ComponentSize.Large}>
+    <EmptyState size={ComponentSize.ExtraSmall}>
       <div className="flow-empty">
         <div className="flow-empty--graphic" />
-        <EmptyState.Text>
+        <EmptyState.Text style={{marginBottom: '0px'}}>
```
Comment (asalem1) [inline src/flows/components/FlowsIndexEmpty.tsx:11]: Can we just apply a class here?

**Comment by asalem1** [comment]
pics or it didn't happen

**Comment by joelfried** [comment]
image

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2680)

**Review by drdelambre — APPROVED** [review]

**Pull request merged**